python/test/t_Gibbs_std.py

```diff
@@ -174,15 +172,15 @@ def getParameter(self):
 )  # samples from Dirac(20)
 # samples from Normal(0,1) x Normal(0,1) x Dirac(20)
 gibbs = ot.Gibbs([normal0_rwmh, normal1_rwmh, dirac_rwmh])
-sample = gibbs.getSample(1000)
+sample = gibbs.getSample(5000)
 recompute = gibbs.getRecomputeLogPosterior()
 print(recompute)
 assert recompute == ot.Indices([1, 0, 1]), "wrong recompute indices"
 mean = sample.computeMean()
 stddev = sample.computeStandardDeviation()
 print(mean, stddev)
-ott.assert_almost_equal(mean, [-0.015835, 0.169951, 20])
-ott.assert_almost_equal(stddev, [0.956516, 1.05469, 0])
+ott.assert_almost_equal(mean, [0.0, 0.0, 20.0], 0.0, 0.1)
```

Comment:
The change looks good to me. Using previous simulations to set reference values is not a unit test, but a regression test. In this case, you state that the reference value is 0.0, with an absolute tolerance set to 0.1.